### PR TITLE
Add pixels perfect test page

### PR DESCRIPTION
**Pull request by vchimev**
Pixels perfect and device independent pixels

### CODE REVIEW

**Comment by vchimev** [comment]
### Preview
iOS:
screen shot 2017-03-15 at 4 55 35 pm

Android:
screen shot 2017-03-15 at 4 55 38 pm

File: apps/app/ui-tests-app/pages/main-page.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export function pageLoaded(args: EventData) {
     examples.set("i61", "pages/i61");
     examples.set("i73", "pages/i73");
     examples.set("i86", "pages/i86");
+    examples.set("asdf", "pages/asdf");
```
Comment (vakrilov) [inline apps/app/ui-tests-app/pages/main-page.ts]: I this on purpose?

Comment (vchimev) [inline apps/app/ui-tests-app/pages/main-page.ts]: Nope, forgotten. Deleted, thanks.

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Comment by vchimev** [comment]
run ci

**Pull request merged**